test.js

```diff
@@ -181,6 +181,21 @@ test('removing regex pattern without other keys is supported', function (t) {
   t.equal(instance.lookup({ to: 'you' }), null)
 })
 
+test('removing regex pattern without other keys is supported - wildcard case', function (t) {
+  t.plan(2)
+
+  var instance = bloomrun()
+  var pattern = { to: /^[a-zA-Z0-9-.]+$/i }
+
+  instance.add(pattern)
```

Comment:
Please add a negative case here to ensure that it doesn't remove all patterns at once.

Reply:
No. I've already spent a lot of time on your code. I don't have any more time. If you know how to make these edits better - do it! I'm done!